cristal-integration-webjar/src/main/node/package.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
No officially part of our best practices but I suggest running `pnpx prettier-package-json --write package.json` for uniformity

Reply:
Done.